From 8171b1b06bae6764164eec7eaf71f8078518a9a2 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:58:30 -0400
Subject: [fs] btrfs: remove unused variables from extent_io.c

Message-id: <1407179109-13714-24-git-send-email-sandeen@redhat.com>
Patchwork-id: 86963
O-Subject: [RHEL7.1 PATCH 023/422] btrfs: remove unused variables from extent_io.c
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Valentina Giusti <valentina.giusti@microon.de>

commit 50892bac3b93afa5cc8de6541a8013a21bef3f74
Author: Valentina Giusti <valentina.giusti@microon.de>
Date:   Mon Nov 4 22:34:25 2013 +0100

    btrfs: remove unused variables from extent_io.c

    Remove unused variables:
    * tree from end_bio_extent_writepage,
    * item from extent_fiemap.

    Signed-off-by: Valentina Giusti <valentina.giusti@microon.de>
    Signed-off-by: Josef Bacik <jbacik@fusionio.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/extent_io.c b/fs/btrfs/extent_io.c
index fe66e54..9f580f2 100644
--- a/fs/btrfs/extent_io.c
+++ b/fs/btrfs/extent_io.c
@@ -2333,13 +2333,11 @@ int end_extent_writepage(struct page *page, int err, u64 start, u64 end)
 static void end_bio_extent_writepage(struct bio *bio, int err)
 {
  struct bio_vec *bvec = bio->bi_io_vec + bio->bi_vcnt - 1;
- struct extent_io_tree *tree;
  u64 start;
  u64 end;
 
  do {
   struct page *page = bvec->bv_page;
-  tree = &BTRFS_I(page->mapping->host)->io_tree;
 
   /* We always issue full-page reads, but if some block
    * in a page fails to read, blk_update_request() will
@@ -4082,12 +4080,10 @@ int extent_fiemap(struct inode *inode, struct fiemap_extent_info *fieinfo,
  struct extent_map *em = NULL;
  struct extent_state *cached_state = NULL;
  struct btrfs_path *path;
- struct btrfs_file_extent_item *item;
  int end = 0;
  u64 em_start = 0;
  u64 em_len = 0;
  u64 em_end = 0;
- unsigned long emflags;
 
  if (len == 0)
   return -EINVAL;
@@ -4112,8 +4108,6 @@ int extent_fiemap(struct inode *inode, struct fiemap_extent_info *fieinfo,
  }
  WARN_ON(!ret);
  path->slots[0]--;
- item = btrfs_item_ptr(path->nodes[0], path->slots[0],
-         struct btrfs_file_extent_item);
  btrfs_item_key_to_cpu(path->nodes[0], &found_key, path->slots[0]);
  found_type = btrfs_key_type(&found_key);
 
@@ -4181,7 +4175,6 @@ int extent_fiemap(struct inode *inode, struct fiemap_extent_info *fieinfo,
    offset_in_extent = em_start - em->start;
   em_end = extent_map_end(em);
   em_len = em_end - em_start;
-  emflags = em->flags;
   disko = 0;
   flags = 0;
 
-- 
1.7.1